src/custom/components/OrderProgressBar/index.tsx

```diff
@@ -363,84 +359,3 @@ function useGetProgressBarInfo({
     isPending: orderIsPending,
   }
 }
-
```

Comment:
Moved this to also to separate file `src/custom/components/TransactionExecutedContent/styled.ts`